test/Streamly/System/Process.hs

```diff
@@ -157,7 +166,8 @@ toBytes2 = monadicIO $ run checkFailAction
     where
 
     action = do
-        S.drain $ Proc.toBytes' executableFileFail []
+        S.drain $
+             Proc.toBytes' interpreterFile [interpreterArg, executableFileFail]
```

Comment:
There is a lot of repetition, we can perhaps use an idiom like we have in streamly-shell.

Reply:
Probably. We can probably abstract stuff out later.